MdeModulePkg/Library/MemoryTypeInfoSecVarCheckLib/MemoryTypeInfoSecVarCheckLib.c

```diff
@@ -84,7 +95,7 @@ MemoryTypeInfoVarCheckHandler (
   if (DataSize != EFI_MEMORY_TYPE_INFORMATION_VARIABLE_SIZE) {
     DEBUG ((
       DEBUG_ERROR,
-      "ERROR: %a() - DataSize = 0x%x Expected = 0x%x\n",
+      "ERROR: %a() - DataSize = 0x%x Expected = 0x%x. Check actual memory types against expected memory types.\n",
       __FUNCTION__,

```

Comment:
should we take this opportunity to switch to __func__?

Reply:
I was thinking we do that in batch across the repo at some point. Will keep it all together and make other changes less tangled with it.

Reply:
Probably as part of the rebase since a lot of changes will already come from the upstream with __func__.